test/e2e-common.sh

```diff
@@ -278,20 +291,22 @@ function install() {
     "${REPO_ROOT_DIR}/test/config/ytt/core"
   )
 
+  local ingress_impl="${GATEWAY_API_IMPLEMENTATION}"
   if is_ingress_class istio; then
     # Istio - see cluster_setup for how the files are staged
     YTT_FILES+=("${E2E_YAML_DIR}/istio/${ingress_version}/install")
   elif is_ingress_class gateway-api; then
-    # This installs an istio version that works with the v1alpha1 gateway api
-    YTT_FILES+=("${E2E_YAML_DIR}/gateway-api/install")
+    # This installs an istio/contour version that works with the v1 gateway api
+    YTT_FILES+=("${E2E_YAML_DIR}/gateway-api/install-${ingress_impl}")
     YTT_FILES+=("${REPO_ROOT_DIR}/third_party/${ingress}-latest/gateway-api.yaml")
     YTT_FILES+=("${REPO_ROOT_DIR}/third_party/${ingress}-latest/net-gateway-api.yaml")
-    YTT_FILES+=("${REPO_ROOT_DIR}/third_party/${ingress}-latest/istio-gateway.yaml")
+    YTT_FILES+=("${REPO_ROOT_DIR}/third_party/${ingress}-latest/${ingress_impl}-gateway.yaml")
   else
     YTT_FILES+=("${REPO_ROOT_DIR}/third_party/${ingress}-latest")
   fi
 
-  YTT_FILES+=("${REPO_ROOT_DIR}/test/config/ytt/ingress/${ingress}")
+  YTT_FILES+=("${REPO_ROOT_DIR}/test/config/ytt/ingress/${ingress}/ingress-class.yaml")
+  YTT_FILES+=("${REPO_ROOT_DIR}/test/config/ytt/ingress/${ingress}/${ingress_impl}")
```

Comment:
I think this is what broke the kind tests for vanilla contour and kourier

Reply:
oh true...